porting/debugging/troubleshooting/system-issues.rst

```diff
@@ -0,0 +1,446 @@
+System-Level Issues
+===================
+
+This part of the guide covers problems with Ubuntu Touch system components, services, and overall functionality. System issues often manifest after successful boot but prevent normal operation.
+
+Quick Diagnosis
+---------------
+1. Check system status::
+
+    # View system logs
+    journalctl -b
+    
+    # Check running services
+    systemctl status
+    
+    # Monitor resource usage
+    top -b -n 1
+
+Understanding these commands:
+
+* journalctl -b: Shows messages from current boot
+* systemctl status: Reports service health
+* top: Reveals resource usage and running processes
+
+2. Verify container state::
+
+    # Check LXC status
+    lxc-info -n android
+    
+    # View container logs
+    journalctl -u lxc@android
+
+Why check the container?
+* Android container provides hardware support
+* Container failure means no hardware access
+* Logs reveal initialization problems
+
+3. Test system access::
+
+    # Check system mounts
+    mount | grep system
+    
+    # Verify important services
+    systemctl status ofono repowerd unity8
```

Comment:
Wrong on 2 accounts:

- `unity8` is now named `lomiri` (and the actual unit name is `lomiri-full-session.service`).
- `lomiri-full-session.service` is a user unit. So you need `systemctl --user`.

Reply:
OK. I'll rewrite that.